### PR TITLE
Pixi: Implement "Take action" texts

### PR DESCRIPTION
**Pull request by caroqliu**
Also cleans up a few related i18n bits. :)

### CODE REVIEW

File: pixi/src/ui/recommendations/RecommendationsView.js [hunk 1]
```diff
@@ -83,14 +83,15 @@ export default class RecommendationsView {
...
+        const count = tagIdCounts[tagId] || 0;
+        tagIdCounts[tagId] = count + 1;
       }
 
       this.recommendationNodes.push(recommendation);
       this.container.appendChild(recommendation);
     }
 
-    for (const tagId of new Set(tagIds).values()) {
+    for (const tagId of Object.keys(tagIdCounts)) {
```
Comment (patrickkettner) [inline pixi/src/ui/recommendations/RecommendationsView.js:94]: why this change?

**Review by patrickkettner — APPROVED** [review]
LGTM!

File: pixi/src/ui/recommendations/RecommendationsView.js (hunk 1 above)
Comment (caroqliu) [reply pixi/src/ui/recommendations/RecommendationsView.js:94]: The Set was used to get unique values from an array with duplicates. I replaced the array with duplicates with an object that keeps count of values instead.
